src/redux/modules/assets.js

```diff
@@ -77,7 +77,7 @@ const onFetchListFailure = (state, action) => {
   }));
 };
 
-const onSSRPrepare = (state, action) => {
```

Comment:
is its relevant to use onSSRPrepare without SSR HYDRATE?

Reply:
Have no idea. This PR's aim is only to produce the same behavior with RTK as it was before RTK.